scripts/generate_unpaved_patterns.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
```

Comment:
This would need porting to python3

Reply:
Yep. I'm porting this to Python 3. At the same time, I will suppress the C code (helper programm linking against littleCMS) and do everything within the Python script using colormath, which we are using yet in other scripts. So almost same result with much less code. For some colors (motorway-fill primary-fill primary-low-zoom secondary-fill trunk-fill trunk-low-zoom), colormath gives a mimimal different result (+1 or -1 within the RGB range of 0..255), but nothing that would be visible.